src/com/google/javascript/jscomp/CommandLineRunner.java

```diff
@@ -1064,6 +1063,7 @@ private void initConfigFromFlags(String[] args, PrintStream out, PrintStream err
       // For backwards compatibility, allow both commonJsPathPrefix and jsModuleRoot.
       List<String> moduleRoots = new ArrayList<>(flags.commonJsPathPrefix);
       moduleRoots.addAll(flags.moduleRoot);
+      moduleRoots.add(ES6ModuleLoader.DEFAULT_FILENAME_PREFIX);
```

Comment:
*If* we can remove it from here, `ES6ModuleLoader.DEFAULT_FILENAME_PREFIX` itself eventually can be removed from the entire project. I think I should look a little more deeply it.